doc/source/io.rst

```diff
@@ -3078,8 +3062,7 @@ any pickled pandas object (or any other pickled object) from file:
 
 .. warning::
 
-   Several internal refactorings, 0.13 (:ref:`Series Refactoring <whatsnew_0130.refactoring>`), and 0.15 (:ref:`Index Refactoring <whatsnew_0150.refactoring>`),
-   preserve compatibility with pickles created prior to these versions. However, these must
```

Comment:
I would actually leave something this warning because this also applies for 0.19 to current

Reply:
I've looked through the ``what's new`` section and can find some changes in pickling but can't find anything new about incompatibility. Could you guide me to the relevant section?

Reply:
@jreback , I've made a proposal on thids warning, that is more general in nature. If you pictured something different, let me know.

Reply:
http://pandas.pydata.org/pandas-docs/stable/whatsnew.html#reorganization-of-the-library-privacy-changes

generally for pickle you should simply use pd.read_pickle as we can preserve back compat

Reply:
Ok, I've written it anew